### ISSUE TITLE
Fix border and margin issue with area below gridTransactionDetails

### ISSUE THREAD

**Issue by Mi-Lan** [protofire/omen-exchange#1484]
- Seems that from time to time borders are missing in this area because of the current layout:
![Screenshot 2021-01-04 at 15 09 27](https://user-images.githubusercontent.com/33226956/103543457-d927de00-4e9e-11eb-9fd6-bb206446e6ee.png)
- Problem seem to be fact that we add borders based on other elements present but i would suggest that we implement flexbox with row-gap so that in the future when we decide to add some new window we don't have to worry about this and they will just stack on top of each other.

**Comment by Mi-Lan** [comment]
@KadenZipfel what do you think about this?

**Comment by KadenZipfel** [comment]
I agree, the current setup of manually setting margins/borders based on certain conditions is far from ideal
